frontend/components/badge-examples.js

```diff
@@ -54,25 +58,32 @@ Badge.propTypes = {
   onClick: PropTypes.func.isRequired,
 };
 
-const Category = ({ category, examples, baseUri, longCache, onClick }) => (
-  <div>
-    <h3 id={category.id}>{ category.name }</h3>
-    <table className="badge">
-      <tbody>
-        {
-          examples.map(badgeData => (
-            <Badge
-              key={badgeData.key}
-              {...badgeData}
-              baseUri={baseUri}
-              longCache={longCache}
-              onClick={onClick} />
-          ))
-        }
-      </tbody>
-    </table>
-  </div>
-);
+const Category = ({ category, examples, baseUri, longCache, onClick }) => {
+  if (examples.filter(example => example.shouldDisplay()).length === 0){
+    return null;
+  }
```

Comment:
In a future pass it might be good to remove the `shouldDisplay` thing. That was kind of a hack to support hiding the rows in CSS, which we're no longer doing. Rendering the filtered examples would be cleaner.